### DISCUSSION TITLE
Change Background of inline code statements (backticks)

### DISCUSSION THREAD

**Discussion by Blackjacx** [WWDCNotes/WWDCNotes#49] [Q&A]
Hey ✌️ 

I wondered if it is possible to change the background of inline code statements to lets say gray like the GitHub style markdown is doing it. I was not able to find a DocC Website that has this :-( 

I'm [building my own](https://blackjacx.github.io/Me) and inline code is not very legible... If we find a solution you could use it to improve legibility of WWDCNotes too.

**Comment by Jeehut** [comment]
I would love to see inline code statements to have a different background color to make them stand out more. But I'm not sure if this is something that should be discussed here as it's not directly related to WWDC Notes but needs to be improved within DocC itself.

The good news is that DocC is open-source and you can bring this up as an issue in the [GitHub repo](https://github.com/apple/swift-docc) or the [Swift froums section](https://forums.swift.org/c/development/swift-docc/80).

**Reply by Blackjacx** [reply]
Thx I'll open an issue there 👍
